q2_diversity/plugin_setup.py

```diff
@@ -217,6 +217,27 @@
     description=("Apply principal coordinate analysis.")
 )
 
+plugin.methods.register_function(
+    function=q2_diversity.pcoa_biplot,
+    inputs={'pcoa': PCoAResults,
+            'features': FeatureTable[RelativeFrequency]},
+    parameters={},
+    outputs=[('biplot', PCoAResults % Properties('biplot'))],
+    input_descriptions={
+        'pcoa': 'The PCoA where the features will be projected onto.',
+        'features': 'Variables to project onto the PCoA matrix'
+    },
+    parameter_descriptions={},
+    output_descriptions={'biplot': 'The resulting PCoA matrix.'},
+    name='Principal Coordinate Analysis Biplot',
+    description="Project features into a principal coordinates matrix. The "
+                "features used can be environmental variables or the features"
```

Comment:
Should the description skip the environmental variables, or should we add support for it in this action?

It does seem that the API get's a little tricky since metadata would be a parameter. (would both become optional and mutually exclusive?)

Reply:
Indeed it gets tricky. I've opted to remove that for now. Later, when the API has been exercised a bit more, we can discuss of better ways to do this.